tests/unit/test_utils.py

```diff
@@ -32,3 +40,35 @@ def async_return(result: t.Any) -> asyncio.Future:
     f = asyncio.Future()
     f.set_result(result)
     return f
+
+
+def mock_sqlalchemy(data: list = []) -> None:
+    """
+    Mock the SQLAlchemy session for unit tests.  Be sure to call unmock_sqlalchemy() in the teardown_method of the test to reset mocks.
+
+    See documentation for mock_alchemy at https://mock-alchemy.readthedocs.io/en/latest/user_guide/index.html#stub-data
+    for more information on how to pass data in.
+
+    Returns: The session mock object.  This can be used in the test to further configure the mock (e.g. to add objects)
```

Comment:
I'd recommend using the reStructuredText (rst) format for descriptions in functions and methods, as this has been the format used throughout the project since its beginning. The one that you're using is google docstring style. I personally don't have any preference to any of them, but just the fact that `rst` is already there, just makes it more reasonable to use.